region/wreckedship/main.json

```diff
@@ -2113,6 +2159,47 @@
         {
           "from": 1,
           "to": [
+            {
+              "id": 1,
+              "strats": [
+                {
+                  "name": "Kihunters and Atomics Farm (Power On)",
+                  "notable": false,
+                  "requires": [
+                    "f_DefeatedPhantoon",
+                    {"or": [
+                      "ScrewAttack",
+                      "Plasma",
+                      "Wave"
+                    ]},
+                    {"resetRoom": {
+                      "nodes": [1, 2, 3],
+                      "mustStayPut": false
+                    }},
+                    {"refill": ["Energy", "Missile", "Super"]}
+                  ]
+                },
+                {
+                  "name": "Atomics and Coverns Farm (Power Off)",
+                  "notable": false,
+                  "requires": [
+                    {"not": "f_DefeatedPhantoon"},
+                    {"or": [
+                      "Plasma",
```

Comment:
Spazer works really well.
Charge is a little slow maybe adding ice makes kill times better?
Morph bombs is good

I use this farm often with just power beam but you need to do moving shots.  I'm a little interested in adding a canMovingShot tech.

Speed echoes are a cool way to farm.  Horizontal spark into the mid height ones.  The ceiling drops you need to `momentumConservingTurnaround` to clip slightly into the glass off center to get the drops, but its not required to get those drops for this.

Reply:
Charge seems too slow, but ice alone seems fine.

Reply:
I just assumed ice alone would work like power beam 🤔